src/main/java/svc/data/textMessages/SMSSpamProtection.java

```diff
@@ -0,0 +1,54 @@
+package svc.data.textMessages;
+
+import java.util.Timer;
+import java.util.TimerTask;
+
+import javax.servlet.http.HttpSession;
+
+public class SMSSpamProtection {
```

Comment:
This class is not a domain class but is named as such. This thing isn't "protection" but rather it is a "protector"